test/pgtest/copy.pt

```diff
@@ -57,3 +57,20 @@ CopyData "[255, 255]"
 CopyDone
 CommandComplete {"tag":"COPY 4"}
 ReadyForQuery {"status":"I"}
+
+# Verify CSV output.
+send
+Query {"query": "COPY (VALUES (1, '2'), (3, '4'), (5, '\\\t\n\rtest\\N'), (6, NULL) ORDER BY column1) TO STDOUT WITH (FORMAT csv)"}
```

Comment:
add a comma to the bad string. also did you run this against postgres to make sure it's identical?

Reply:
Ah no I didn't - I just ran this test case using the `mzcompose --find cargo-test` command:
https://github.com/MaterializeInc/materialize/blob/c02dfbdf9f47f7a456f79123d5a5daa61d0fa02a/src/environmentd/tests/pgwire.rs#L524-L528

and assumed that it tests against both PG and MZ. If it's a manual process I can run through that now.

Reply:
Okay, adding commas to both tests and confirmed this test passes locally using `mz-pgtest` against a local postgres on 15.6.

Reply:
Sadly we don't auto run these against postgres yet because of very annoying version skew errors we currently have (some files only work against specific postgres versions).